cumulus/test/runtime/src/lib.rs

```diff
@@ -293,6 +356,11 @@ construct_runtime! {
 		TransactionPayment: pallet_transaction_payment,
 		TestPallet: test_pallet,
 		Glutton: pallet_glutton,
+		Authorship: pallet_authorship,
+		CollatorSelection: pallet_collator_selection,
+		Session: pallet_session,
```

Comment:
Same for these two. We have hard coded authorities any way? Aka they are not changing.